src/olympia/addons/tasks.py

```diff
@@ -562,6 +563,19 @@ def add_static_theme_from_lwt(lwt):
     for addon_tag in AddonTag.objects.filter(addon=lwt):
         AddonTag.objects.create(addon=addon, tag=addon_tag.tag)
 
+    # Steal the ratings (even with soft delete they'll be deleted anyway)
+    addon_updates.update(
+        average_rating=lwt.average_rating,
+        bayesian_rating=lwt.bayesian_rating,
+        total_ratings=lwt.total_ratings,
+        text_ratings_count=lwt.text_ratings_count)
+    for rating in lwt.ratings.all():
+        rating.update(addon=addon, version=version)
+        # Recreate the activity log entry so it's there in devhub.
+        activity.log_create(
+            amo.LOG.ADD_RATING, addon, rating, user=rating.user,
+            created=rating.created)
```

Comment:
Shouldn't we rewrite the existing activity instead?

Reply:
We could, but historically we've never edited existing log entries... and it's encoded as JSON in the DB so there's a chance something will be mangled.  
On the other hand, the flaw with adding a new log is we don't replicate `EDIT_RATING` events and the difference between `Rating.user` and `Rating.user_responsible` is concerning to me.
I'm conflicted.

Reply:
It's annoying to rewrite, but I think it's the right thing to do, otherwise we'll end up with duplicate logs, pointing to an add-on that no longer exists.

Reply:
https://github.com/mozilla/addons-server/pull/8403/commits/8f7d8ab1ea90d306e4b788dc7d0dd543fcbaefcf - a lot more complicated than it seemed because ActivityLog doesn't support indexing by Rating so I had to do all review-type actions for that Addon.